### DISCUSSION TITLE
What term describes a person who enjoys solving/creating crosswords 🧩?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1088] [Galaxy Trivia] [answered]
**Bonus:** A pangram is a sentence containing what?

**Comment by peterbaccaro** [comment]
The term **cruciverbalist** describes a person who enjoys solving/creating crosswords puzzles.

A pangram is a sentence **that includes every letter of the alphabet, A through Z**.

**Comment by NomanGhiasy** [comment]
The meaning of CRUCIVERBALIST is a person skillful in creating or solving crossword puzzles.

**Comment by thuongtruong1009** [comment, marked as answer]
A person who enjoys solving or creating crosswords is called a "cruciverbalist."

A pangram is a sentence containing every letter of the alphabet at least once. It is also known as a "holoalphabetic sentence." One of the most famous pangrams in the English language is "The quick brown fox jumps over the lazy dog."
